src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashRepository.java

```diff
@@ -231,6 +231,10 @@ public boolean mergePullRequest(String pullRequestId, String version) {
     return this.client.mergePullRequest(pullRequestId, version);
   }
 
+  public void markStatus(String pullRequestId, String status) {
+    this.client.markStatus(pullRequestId, status);
```

Comment:
`this` is not needed. The existing code overuses `this`, but it's not a good example to follow IMHO.

Reply:
done